aspnetcore/getting-started-1.1.md

```diff
@@ -23,9 +23,6 @@ uid: getting-started-1.1
 
 1. Install [.NET Core](https://microsoft.com/net/core).
 
-   If you're on Windows, select the **Command line / other** environment. 
```

Comment:
I think it's best to decouple ourselves from a installation experience for .NET Core. Any description or images that we provide for installing .NET Core runs the risk of going stale or out of date as the .NET site gets updated. All we should do is tell people to install .NET Core and provide a link.